app/index.js

```diff
@@ -2,21 +2,21 @@ import React, { View, Text } from 'react-native';
 
 import styles from './styles/default';
 
-class TwoByTwoCards extends React.Component {
+const TwoByTwoCards = React.createClass({
   render() {
     return (
       <View style={[styles.container, { flexDirection: 'row' }]}>
         <View style={{ flex: 1 }}>
-          <View style={{ flex: 1 }}><Text>One</Text></View>
-          <View style={{ flex: 1 }}><Text>Two</Text></View>
+          <View style={styles.matrixInner}><Text>One</Text></View>
+          <View style={styles.matrixInner}><Text>Two</Text></View>
         </View>
-        <View style={{ flex: 1 }}>
-          <View style={{ flex: 1 }}><Text>Three</Text></View>
-          <View style={{ flex: 1 }}><Text>Four</Text></View>
+        <View style={{flex: 1}}>
```

Comment:
**<a href='http://eslint.org/docs/rules/object-curly-spacing'>object-curly-spacing</a>**: A space is required after '{'